data/academic_institutions/maryland.md

```diff
@@ -8,7 +8,7 @@ continent: North America
 courses:
     - name: Organization of Programming Languages-(along with Ruby, Prolog, Java) 
       acronym: CMSC 330
-      online_resource: "https://www.cs.umd.edu/class/fall2014/cmsc330/"
+      online_resource: "https://www.cs.umd.edu/class/fall2014/cmsc330/ - [403 Forbidden]"
```

Comment:
It seems that the URL of this course changes every semester. https://github.com/umd-cmsc330 contains some materials.

Reply:
Thank you, that's a much more useful link!